framework/src/modules/http_api/schema/swagger.yml

```diff
@@ -2208,13 +2220,13 @@ definitions:
           The delegates username.
           A delegate chooses the username by registering a delegate on the Lisk network.
           It is unique and cannot be changed later.
-      voteWeight:
+      totalVotesReceived:
         type: string
         example: 1081560729258
         description: |
-          The voters weight of the delegate.
-          Represents the total amount of Lisk (in Beddows), that the delegates voters own.
-          The voters weight decides which rank the delegate gets in relation to the other delegates and their voters weights.
+          The total votes received by the delegate.
```

Comment:
```suggestion
          The total votes received of the delegate.
```
?

Reply:
@shuse2 are you sure `of` is correct? this are the votes the delegate received

Reply:
Im not sure =)
but those votes are not necessarily  voted by delegate, but it sounds like it